docs/EN_US/HPCCSpark/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+################################################################################
+get_filename_component(DOC_DIR_NAME ${CMAKE_CURRENT_SOURCE_DIR} NAME)
+include(${CMAKE_CURRENT_SOURCE_DIR}/../../BuildTools/cmake_config/${DOC_DIR_NAME}.txt)
```

Comment:
It might be a good idea to build your path based on  ${HPCC_SOURCE_DIR} rather than creating a relative path

Reply:
I will defer to @xwang2713 on the build components.

Reply:
@g-pan  you're mixing relative and absolute paths, Please see https://github.com/hpcc-systems/HPCC-Platform/blob/master/docs/XMLGeneration/CMakeLists.txt as an example